api/accurate/v2alpha1/conversion_test.go

```diff
@@ -24,6 +24,6 @@ func SubNamespaceStatusFuzzFunc(_ runtimeserializer.CodecFactory) []interface{}
 	}
 }
 
-func SubNamespaceStatusFuzzer(in *SubNamespace, c fuzz.Continue) {
-	c.FuzzNoCustom(in)
+func SubNamespaceStatusFuzzer(in *SubNamespace, c randfill.Continue) {
+	c.Fill(in)
```

Comment:
```suggestion
	c.FillNoCustom(in)
```

Ref. https://github.com/cybozu-go/accurate/pull/159/files#r2181270241

Reply:
fixed [6183777](https://github.com/cybozu-go/accurate/pull/159/commits/618377728b0b07451533bf045b12c5c490d44f12)